docs/migration/migratingtonetcode.md

```diff
@@ -660,13 +660,13 @@ public class DamageClass : NetworkBehaviour
 
 ## Network Discovery
 
-Netcode does not provide Network Discovery. The Contributions repository provides an example implementation for [NetworkDiscovery] (https://github.com/Unity-Technologies/multiplayer-community-contributions/tree/main/com.community.netcode.extensions/Runtime/NetworkDiscovery).
+Netcode does not provide Network Discovery. The Contributions repository provides an example implementation for [NetworkDiscovery](https://github.com/Unity-Technologies/multiplayer-community-contributions/tree/main/com.community.netcode.extensions/Runtime/NetworkDiscovery).
```

Comment:
Lobby and matchmaking sort of does this though no?

Reply:
That I will look into more. Initially, I was just fixing the broken link from the spacing issue.

Reply:
I have another task to go through all the installation/migration/upgrade guides soon. I'll note this for that task.

Reply:
Relevant task: https://jira.unity3d.com/browse/MTT-3089